docs/src/modules/components/ApiPage.js

```diff
@@ -140,10 +140,10 @@ export default function ApiPage(props) {
       inheritance,
       themeDefaultProps: pageContent.themeDefaultProps,
     }),
-    ...getCssToC({
+    ...getClassesToC({
       t,
       componentName: pageContent.name,
-      componentStyles,
+      componentClasses,
     }),
     componentSlots?.length > 0 && createTocEntry('slots'),
     hasClasses && createTocEntry('classes'),
```

Comment:
With this modification, you will get a ToC with 

- classes (expanded)
- slots
- classes

Should replace the `hasClasses && createTocEntry('classes'),` by `...getClassesToC({ })` to respect the page order

Reply:
Good catch, thanks! 
It won't actually ever call `createTocEntry('classes')` because I introduced a bug in `hasClasses` ;)